src/Illuminate/Database/Eloquent/Hydrator.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Illuminate\Database\Eloquent;
+
+use Illuminate\Contracts\Database\Eloquent\Hydratable;
+use Illuminate\Contracts\Database\Eloquent\Hydrator as HydratorInterface;
+
+class Hydrator implements HydratorInterface
+{
+    /**
+     * The original model instance.
+     *
+     * @var Model
+     */
+    protected $model;
+
+    /**
+     * A custom database connection.
+     *
+     * @var string
+     */
+    protected $connection;
+
+    /**
+     * Hydrator constructor.
+     * @param Model $model
+     */
+    public function __construct(Model $model)
+    {
+        $this->model = $model;
+    }
+
+    /**
+     * Fill a new Eloquent model instance with raw attributes returned from the query builder.
+     *
+     * @param array $attributes
+     * @return Hydratable
+     */
+    public function hydrate(array $attributes = []) : Hydratable
```

Comment:
the framework does not use return type annotations

Reply:
I removed it. Thanks